### PR TITLE
2D U8 String serialization to gRPC TFS/KServe

### PR DESCRIPTION
**Pull request by dkalinowski**
With tensor name suffix _string_2d_u8 hint

### CODE REVIEW

File: src/tensorinfo.cpp [hunk 1]
```diff
@@ -23,9 +23,12 @@
 
 #include "kfs_frontend/kfs_grpc_inference_service.hpp"
 #include "logging.hpp"
+#include "stringutils.hpp"
 
 namespace ovms {
 
+const std::string STRING_SERIALIZATION_HINT_NAME_SUFFIX = "_string_2d_u8";
```
Comment (dkalinowski) [inline src/tensorinfo.cpp:30]: change to `_string`

**Review by atobiszei — APPROVED** [review]

File: src/deserialization.hpp [hunk 2]
```diff
@@ -356,7 +356,7 @@ Status deserializePredictRequest(
             ov::Tensor tensor;
 
             if (requiresProcessing(requestInput)) {
```
Comment (michalkulakowski) [inline src/deserialization.hpp:358]: If you changed getProcessingHint to getPreProcessingHint maybe change also requiresProcessing to requiresPreProcessing

**Review by michalkulakowski — APPROVED** [review]

File: src/tensorinfo.cpp (hunk 1 above)
Comment (dkalinowski) [reply src/tensorinfo.cpp]: done

File: src/deserialization.hpp (hunk 2 above)
Comment (dkalinowski) [reply src/deserialization.hpp]: done

**Pull request merged**